.github/workflows/visual-tests.yml

```diff
@@ -0,0 +1,85 @@
+name: Visual Tests
+
+on: 
+  workflow_dispatch:
+    inputs:
+      update-snapshots:
+        description: "Update snapshots?"
+        type: boolean
+  schedule:
+    - cron: '59 23 * * *'
```

Comment:
Instead of running on a separate schedule, you could reference this workflow in the nightly workflow file, like [these](https://github.com/shopware/shopware/blob/trunk/.github/workflows/nightly.yml#L122-L149).